TWLight/resources/filters.py

```diff
@@ -11,7 +11,8 @@ class PartnerFilter(django_filters.FilterSet):
 
     tags = django_filters.ChoiceFilter(
         # Translators: On the MyLibrary page (https://wikipedialibrary.wmflabs.org/users/my_library), this text is shown to indicate how many subject areas a collection covers.
-        label=_("Tags"),
+        #Reword 'Tags' to 'Topics'
```

Comment:
There's no need for this comment and it interferes with the translator comment on the line above, so you can safely remove it :)